code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -354,6 +354,7 @@ var/list/gear_datums = list()
 	var/custom_setup_proc  //Special tweak in New
 	var/category
 	var/list/gear_tweaks = list() //List of datums which will alter the item after it has been spawned.
+	var/implanted = FALSE //is this item implanted? Used for augments / Implants.
```

Comment:
Would suggest an `implant_into` var that defaults to `BP_CHEST` instead of a bool and only ever initially targeting chest.